src/validation.cpp

```diff
@@ -2101,6 +2101,13 @@ bool CChainState::FlushStateToDisk(
             nLastFlush = nNow;
             full_flush_completed = true;
         }
+        TRACE6(utxocache, flush,
```

Comment:
This tracepoint is being called each time `FlushStateToDisk()` is called, even when we don't flush. We call `FlushStateToDisk()` with `FlushStateMode::PERIODIC` in `AcceptToMemoryPoolWithTime()` for each transaction.

Reply:
If this is specifically about flushing the UTXO set, it should probably be moved a few lines higher into the conditional where the associated `LOG_TIME_MILLIS...` call is: https://github.com/bitcoin/bitcoin/blob/master/src/validation.cpp#L2087-L2089

However based on the included doc, it sounds like this tracepoint is attempting to include both UTXO cache flushes and block index flushes. I think this will be difficult to do with a single tracepoint, so probably the scope should be reduced to a tracepoint for writing the coins cache to disk.